tests/warn/tostring-interpolated.scala

```diff
@@ -0,0 +1,36 @@
+//> using options -Wtostring-interpolated
```

Comment:
Would it be of value to add a check file here?

Reply:
The one strange message is now in the neg test.